scaaml/capture/ascad/scope.py

```diff
@@ -0,0 +1,131 @@
+"""Context manager for the scope."""
```

Comment:
Something I only noticed now: all the files owned by Google must start with the license header with the correct years mentioned.
Not blocking this PR as I think other files outside of this PR already have been merged but please do a PR to fix it.